.stylelintignore

```diff
@@ -0,0 +1,2 @@
+build
+target
```

Comment:
We have a few artifact directories:

```
dist
docs
es
lib
test-env
types
```

Additionally, would `node_modules` be a good choice to be here?